components/steps/Status.tsx

```diff
@@ -4,8 +4,6 @@ import styled from "styled-components";
 
 import Router from "next/router";
 
-import getProfileStage from "../../lib/getProfileStage";
```

Comment:
unrelated to scope of PR but time to clean up code

Reply:
I feel ya, wanna do a real spring cleaning once this thing goes live lol